src/pages/documentation/adobe-analytics-mobile-services/release-notes.md

```diff
@@ -32,34 +32,12 @@ Release notes and change logs for the Adobe Mobile Services extension
 
 * Released the brand new Adobe Experience Platform Mobile Services iOS Swift SDK.
 
-## April 14, 2021
-
-### iOS Mobile Services 1.1.1
-
-* Fixed a crash that could happen while downloading remote assets.
-
-## December 18, 2020
-
-### iOS Mobile Services 1.1.0
-
-* The AEP SDKs are now distributed using XCFrameworks in order to support hardware with the new Apple M1 architecture while maintaining support for existing Intel architecture.
-  * **IMPORTANT**: Upgrading to XCFrameworks distribution requires Xcode 12.0 or newer
-  * **IMPORTANT**: If using Cocoapods, upgrading to the XCFrameworks distribution requires Cocoapods 1.10.0 or newer
```

Comment:
Should this be removed? Since this mentions the AEP SDKs and this one is still the AEP repo.

Reply:
this 1.1.0 version still refers to acp